quint/cli-tests.md

```diff
@@ -182,11 +182,6 @@ Temporarily disabled.
 <!-- !test check LamportMutex - Types & Effects -->
     quint typecheck ../examples/classic/distributed/LamportMutex/LamportMutex.qnt
 
-### OK on parse counters
-
-<!-- !test check counters -->
-    quint parse ../examples/language-features/counters.qnt
-
 ### OK on parse records
 
 <!-- !test check records -->
```

Comment:
why do we need the tests below? Would not they get tested by running the script in the examples?

Reply:
Missed those, thanks! I've picked them up now.